mesonbuild/dependencies/misc.py

```diff
@@ -301,13 +301,51 @@ def modname_from_filename(self, filename):
             modname = modname[3:]
         return modname
 
+    def compiler_tag(self):
+        tag = None
+        if mesonlib.for_windows(self.want_cross, self.env):
+            comp_ts_version = self.env.detect_cpp_compiler(self.want_cross).get_toolset_version()
+            compiler_ts = comp_ts_version.split('.')
+            #FIXME - what about other compilers?
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->